R/pkg/R/DataFrame.R

```diff
@@ -998,33 +998,39 @@ setMethod("unique",
 #' sparkR.session()
 #' path <- "path/to/file.json"
 #' df <- read.json(path)
+#' collect(sample(df, fraction=0.5))
 #' collect(sample(df, FALSE, 0.5))
-#' collect(sample(df, TRUE, 0.5))
+#' collect(sample(df, TRUE, 0.5, seed=3))
```

Comment:
I think the style here is to have one space between the param name and its value, like `seed = 3` and `fraction = 0.5` above

Reply:
D'oh.